### PR TITLE
replace unit conversion functions with madrat tool wrappers

### PR DESCRIPTION
**Pull request by fbenke-pik**

### CODE REVIEW

File: DESCRIPTION
Comment (johanneskoch94) [inline DESCRIPTION:1]: Does GDPuc now have to be listed under Depends? I don't know..

File: R/EDGE-Industry.R
Comment (johanneskoch94) [inline R/EDGE-Industry.R:1]: And calling a tool function with `::` doesn't hinder the caching right?

**Review by johanneskoch94 — COMMENTED** [review]
Apart from the above comments, it looks good. (Some indentation was also messed up, but that can wait.)

File: DESCRIPTION
Comment (fbenke-pik) [reply DESCRIPTION:1]: As far as I can tell, no

File: R/EDGE-Industry.R
Comment (fbenke-pik) [reply R/EDGE-Industry.R:1]: It should not.

**Review by johanneskoch94 — APPROVED** [review]

**Pull request merged**